docs/source/using/managing-minishift.adoc

```diff
@@ -53,6 +53,32 @@ the GitHub issue https://github.com/minishift/minishift/issues/179[#179].
 The link:../command-ref/minishift_delete{outfilesuffix}[`minishift delete`] command deletes the OpenShift cluster,
 and also shuts down and deletes the Minishift VM. No data or state are preserved.
 
+[[choosing-vm]]
+== Choosing the VM image
```

Comment:
> We should put the exact command in the documentation, so that user can copy paste and run the command.

+1 that would be my preferred option for now. We can make this partly dynamic by making the CentOS ISO a variable. We will need this most likely for other things as well later.